src/components/MyNypl/MyNypl.js

```diff
@@ -167,8 +154,8 @@ MyNypl.propTypes = {
 MyNypl.defaultProps = {
   className: 'myNypl',
   lang: 'en',
-  loginCatalogLink: appConfig.loginMyNyplLinks.catalog,
-  loginResearchLink: appConfig.loginMyNyplLinks.research,
+  loginCatalogLink: appConfig.myNyplLinks.catalog,
+  loginResearchLink: appConfig.myNyplLinks.research,
```

Comment:
Is this the right prop name? Shouldn't it be `catalogLink` and `researchLink` based on what `renderLoginLinks` is using above?

Reply:
Right, these are not the links used for the update. I actually forgot why I made this change... it might've been out of cautiousness so that no login links are used at all.

Reply:
Oh I see! I missed that the right props are used below.